specs/fulu/das-core.md

```diff
@@ -118,6 +131,26 @@ def get_custody_groups(node_id: NodeID, custody_group_count: uint64) -> Sequence
     return sorted(custody_groups)
 ```
 
+### `get_max_blobs_per_block`
+
+```python
+def get_max_blobs_per_block(epoch: Epoch) -> uint64:
+    """
+    Return the maximum number of blobs that can be included in a block for the current epoch.
+    """
+    # This is a temporary block of code which is only necessary for testing. The testing
+    # infrastructure uses states where the current epoch is zero but the current version
+    # is whichever fork is being tested. This does not play well with a getter function
+    # which uses the current epoch to determine the appropriate return value.
+    if epoch < FULU_FORK_EPOCH:
+        return BLOB_SCHEDULE[-1]["MAX_BLOBS_PER_BLOCK"]
```

Comment:
So given this explanation i'd expect that the test config is correctly setting the current fork, and would have to also configure at least a single entry in this array?

I could see returning the maximum as a default being problematic if we're not testing fulu . eg. deneb needs less blobs or the comparison would fail due to structure issues iiuc (using 9 blobs instead of 6 etc)

Reply:
Yes, this wouldn't have worked pre-fulu. I originally had a deneb+ solution, this:
```python
current_version = state.fork.current_version
if get_fork_epoch(current_version) == FAR_FUTURE_EPOCH:
    if current_version == DENEB_FORK_VERSION:
        return MAX_BLOBS_PER_BLOCK
    if current_version == ELECTRA_FORK_VERSION:
        return MAX_BLOBS_PER_BLOCK_ELECTRA
    if current_version >= FULU_FORK_VERSION:
        return BLOB_SCHEDULE[-1]["MAX_BLOBS_PER_BLOCK"]
```
But since this function is currently fulu-only, I thought that was too much. If/when we backport this to deneb, we can deal with that then. I've updated this function once again to just fallback to the minimum value.